enhancements/machine-api/control-plane-machine-set.md

```diff
@@ -754,6 +770,19 @@ discover during testing something that is currently unknown to us. If we discove
 on a particular naming scheme for the Control Plane Machines, we will need to solve this issue within the other
 component.
 
+#### Customizing Control Plane Machine Names
+
+The `MachineNamePrefix` field allows users to customize the naming format of Control Plane Machines managed by the `ControlPlaneMachineSet`. This feature is gated behind the `CPMSMachineNamePrefix` feature gate and must be enabled before use.
+
+When the `MachineNamePrefix` is specified, and `CPMSMachineNamePrefix` feature gate is enabled, it replaces the default naming convention specified in [Naming of Control Plane Machines owned by the ControlPlaneMachineSet](#naming-of-control-plane-machines-owned-by-the-controlplanemachineset).
+The generated machine name will consist of the provided prefix, followed by a randomly generated string of 5 characters, and the machine index.  For example, if the `MachineNamePrefix` is set to `my-control-plane`, a machine name might be `my-control-plane-abcde-0`.
+
+The `MachineNamePrefix` field is subject to the following validation rules:
+
+* It must be between 1 and 32 characters in length.
+* It must only contain lowercase alphanumeric characters and hyphens (`-`).
+* Hyphens are only permitted between characters (i.e. the prefix cannot start or end with a hyphen).
```

Comment:
In the previous part https://github.com/openshift/enhancements/pull/1714/files#diff-167c14da636a95920dc7f720fd0d0be7490305b02914e4113ee9aebf2f51a3bdR119-R122 , it says consisting of `lowercase alphanumeric characters, '-', or '.'`  `between 1 and 245 characters in length.` but here is `1 and 32 characters` `lowercase alphanumeric characters and hyphens (`-`)`, which is inconsistent, is this right?

Reply:
Got it ,thank you!
LGTM